### PR TITLE
* Cleanup redundant usage of `getarg()` in scripts

### PR DESCRIPTION
**Pull request by cydh**
* `npc:enchan_mal.txt`

Signed-off-by: Cydh Ramdh <cydh@pservero.com>

### CODE REVIEW

File: npc/re/merchants/enchan_mal.txt [hunk 1]
```diff
@@ -327,13 +327,15 @@ L_Socket:
 		mes "^0000ff"+.@equip_name$+"^000000! Do you want to enchant this equipment? How will you pay for this?";
 		next;
 
+		.@enclimit = getarg(1);
```
Comment (dastgir) [inline npc/re/merchants/enchan_mal.txt]: getarg(0) not 1

Comment (dastgir) [inline npc/re/merchants/enchan_mal.txt]: Oops, for .@enccost its getarg(0)

File: npc/re/merchants/enchan_mal.txt [hunk 2]
```diff
@@ -327,13 +327,15 @@ L_Socket:
...
+		.@enccost = getarg(1);
```
Comment (dastgir) [inline npc/re/merchants/enchan_mal.txt]: getarg(0)

Comment (cydh) [inline npc/re/merchants/enchan_mal.txt]: ye darn, copy-paste failure

**Comment by cydh** [comment]
@Lemongrass3110, gonna merge this now. I have no time to check for the other.

**Comment by Atemo** [comment]
You closed it too fast :)

**Comment by cydh** [comment]
Ehh sorry. Imho u're late to "reply first" as confirmation that u'll do or not. :P

**Pull request merged**